docs/statistics.md

```diff
@@ -195,37 +223,29 @@ The total number of packets that failed to be decrypted at the receiver side. Av
 
 ### pktSndFilterExtraTotal
 
-The total number of packet filter control packets supplied by the packet filter (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for sender.
+The total number of packet filter control packets sent by the SRT sender (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for sender.
 
-Packet filter control packets are SRT DATA packets.
+The type of the packet filter control packets is DATA. These packets contain only packet filter control information and do not belong to the actual transmitted data.
 
 The `SRTO_PACKETFILTER` socket option should be enabled (refer to [API.md](API.md)). Introduced in SRT v1.4.0.
 
 ### pktRcvFilterExtraTotal
 
-The total number of packet filter control packets received but not returned by the packet filter
-(refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for receiver.
+The total number of packet filter control packets received by the SRT receiver (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Available for receiver.
```

Comment:
The term "SRT receiver" is kinda ambiguous here. The rule is such that if the packet filter is enabled, then it is the first instance to receive a DATA packet - it then receives every such packet, and then makes a decision. It is not defined from upside what will happen in particular situation - only that a filter is free to do whatever it wants with the incoming packet. After that operation the filter simply supplies **0 or more packets**. What may happen then is that:
* incoming was FEC/CTL packet and the filter didn't rebuild anything - supplies 0 packets
* incoming was data packet, retransmitted, after FEC/CTL packet, which filled in the group gap and allowed 3 packets to be rebuilt (the "vicious square" case) - supplies 4 packets
* incoming was a FEC/CTL packet for a group that lacks exactly 1 packet - supplies 1 packet (although it's not the packet that has been received)

The best description that doesn't risk mistakes or unintentional lies, I think, would be:

The total number of packet filter control packets that have been received by the packet filter. These packets are of DATA packet type, but they only contain a control information for the packet filter and do not turn into any deliverable data packets.